docs/changes/4.4.3.md

```diff
@@ -14,6 +14,7 @@ Release date: `2020-xx-xx`
 - [NXDRIVE-2113](https://jira.nuxeo.com/browse/NXDRIVE-2113): Add a warning on upload when server-side lock is disabled
 - [NXDRIVE-2116](https://jira.nuxeo.com/browse/NXDRIVE-2116): Requests `Invalid byte range` multiple of total binary size
 - [NXDRIVE-2124](https://jira.nuxeo.com/browse/NXDRIVE-2124): Uniformize the name: "Direct Edit"
+- [NXDRIVE-2131](https://jira.nuxeo.com/browse/NXDRIVE-2131): [Direct Edit] Handle HTTP 413 error: Request Entity Too Large
```

Comment:
Here, we already know we are talking about Direct Edit, no need to duplicate the info (but the title on Jira should not be changed):
```suggestion
- [NXDRIVE-2131](https://jira.nuxeo.com/browse/NXDRIVE-2131): Handle HTTP 413 error: Request Entity Too Large
```